core/src/main/java/fr/sncf/osrd/infra/api/tracks/undirected/SpeedLimits.java

```diff
@@ -0,0 +1,42 @@
+package fr.sncf.osrd.infra.api.tracks.undirected;
+
+import com.google.common.collect.ImmutableMap;
+import fr.sncf.osrd.railjson.schema.infra.trackranges.RJSSpeedSection;
+import java.util.Collection;
+import java.util.Set;
+
+public class SpeedLimits {
+
+    private final double defaultSpeedLimit;
```

Comment:
We will have one issue with this.
The field `speed_limit` in RJS is now optional. If it's not specified then we don't have a default speed limit.

I think that the parser fails in this case.

Reply:
It should be parsed as NaN, I've made a small change so that it's handled properly